nb/projects/plots/mandala.md

```diff
@@ -0,0 +1,11 @@
+```JavaScript
+(await readSvg('https://jsxcad.js.org/svg/mandala.svg', { fill: false }))
+  .align('xy')
+  .scale(20)
+  .and(toolpath())
+  .gcode('mandala');
```

Comment:
It's really nice how compact this is for going from a .svg to gcode

Reply:
Yeah, it's getting there. :)